### PR TITLE
Add description field to repository record and display in pui

### PR DESCRIPTION
**Pull request by mark-cooper**
https://archivesspace.atlassian.net/browse/AR-1907

### CODE REVIEW

File: common/db/migrations/094_add_description.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require_relative 'utils'
+
+Sequel.migration do
+
+  up do
+    alter_table(:repository) do
+     TextField :description
```
Comment (lmcglohon) [inline common/db/migrations/094_add_description.rb]: Should this be add_column(:description, TextField)?

Comment (mark-cooper) [reply common/db/migrations/094_add_description.rb]: I followed a precedent:

https://github.com/archivesspace/archivesspace/blob/master/common/db/migrations/044_add_subtitle.rb

Definitely works that way. Docs here show `$type` `$fieldname` in block as valid usage:

http://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html#label-A+Basic+Migration

**Comment by cdibella** [comment]
Since this is a data model change (albeit a small one), can you tell us about the reason/use for this field?

**Comment by mark-cooper** [comment]
@cdibella the request came from Matthew Shellenbarger (MercyHurst). Having multiple public repositories they want to be able to have some descriptive text / an "about" section on the repository page for "patrons" to know the purpose / scope of a repository. Particularly useful when the audience for each repository may not naturally overlap.

**Comment by cdibella** [comment]
Thanks, @mark-cooper - that makes sense. In fact, it makes so much sense that I thought someone must have requested it previously and there would be something this could knock off of our JIRA to-do list. Finding nothing, I've made a new one, AR-1907.

**Comment by sdm7g** [comment]
In public/app/views/repositories/_full_repo.html.erb:
Adding a `<br>` before the About header makes it look nicer (IMHO). (or you could do it in CSS)
Wrapping it with process_mixed_content(  ).html_safe gets it to display paragraph breaks. 
( Some other code I haven't looked at yet required on frontend to display mixed content. ) 

```
<% if info["top"]["description"].present? %>
<br>
<div class="description">
  <p><strong><%= t('description') %>: </strong> <br/><%= process_mixed_content(info["top"]["description"]).html_safe %></p>
</div>
<% end %>
```

**Comment by mark-cooper** [comment]
@sdm7g thanks for the feedback! Personally I'd be inclined to leave things as they are or perhaps call `simple_format` (newlines to p tags, everything else sanitized by default). Besides CSS it's easy to override `public/app/views/repositories/_full_repo.html.erb` in a plugin to cater for local preferences.

@lmcglohon we should probably look into how safe `process_mixed_content($variable).html_safe` is with data from user input (quick grep finds it being used a lot). If javascript can be executed that should be tightened up.

**Comment by sdm7g** [comment]
@mark-cooper Agree: simple_format works for me with fewer issues on the fronted. The main point was to get the paragraph breaks on public display.

**Comment by mark-cooper** [comment]
@sdm7g I added `simple_format` to it :+1:

**Comment by sdm7g** [comment]
+1 @lmcglohon

**Pull request merged**